setup.cfg

```diff
@@ -1,4 +1,4 @@
 [pytest]
 python_files = *.py
-addopts = --verbose --ignore setup.py --ignore utide/robustfit.py --doctest-modules --cov-report term-missing --cov .
```

Comment:
@ocefpaf, regarding coverage reporting: I think it is too early for that; and I admit I have no experience with it, and don't know how it works.  I'm skeptical about the ability of automation to be helpful in this.  I don't want things to get too complicated.  
Regarding doctests, again, I tend to think that putting tests in the docs is making things unnecessarily complicated.  Isn't it simpler to keep all the tests together in the tests directory?  Maybe doctests would be useful after things have settled down, but I suspect that right now they would get in the way.